app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter(post, likes_number)
+    post = Post.where(id: post.id)
+    post.update(likesCounter: likes_number)
+  end
```

Comment:
- [ ] You do not have to set post and likes counter implicitly, rails is smart enough, this will work perfectly: 

```
def update_post_counter
    user.increment!(:posts_counter)
  end
```